VOC2007/ImageSets/Main/aeroplane_train.txt

```diff
@@ -0,0 +1,2501 @@
+000012 -1
```

Comment:
@medo These files are the ones provided by pascal and are needed to know what file belongs to what category. These files have to be in the repo in order to train or test. They need to be merged. Without these files we'll have the 10K images without knowing what image belongs to what category. If you want us to have these files locally without having them on the repo, how is any of you going to test it in order to merge?